modules/javafx.graphics/src/main/java/javafx/scene/text/TextFlow.java

```diff
@@ -59,38 +59,38 @@
 import javafx.css.StyleableProperty;
 
 /**
- * TextFlow is special layout designed to lay out rich text.
- * It can be used to layout several {@link Text} nodes in a single text flow.
- * The TextFlow uses the text and the font of each {@link Text} node inside of it
+ * A special layout designed to lay out rich text.
```

Comment:
What's "special" about it? :-)
```suggestion
 * A layout designed to lay out rich text.
```

Reply:
Maybe it's special in the sense that it's specialized towards displaying text, unlike other `Pane`s that are multi-purpose for all nodes. "designed to" already points to that. Maybe "A specialized layout for rich text."?

Reply:
A specialized layout... +1

Reply:
```suggestion
 * A specialized layout for rich text.
```